package.json

```diff
@@ -15,7 +15,7 @@
     "clean": "turbo clean && rm -rf node_modules",
     "dev": "turbo dev --no-cache --parallel --continue",
     "format": "prettier --write \"**/*.{ts,tsx,md}\"",
-    "lint": "turbo lint",
+    "lint": "eslint packages",
```

Comment:
Isn't the issue with using `wearerequired/lint-action@v2` in the workflow? Would swapping that out for running `yarn lint` solve it?

Reply:
https://github.com/Localitos/pluto/blob/de3bc49785a1b8bd2b8343929c2d34298aa69b0d/.github/workflows/on_pull_request.yml#L153-L158

Reply:
Wouldn't it be better to get linting working with Turbo so that we can take advantage of the paralleziation and caching benefits?

It looks like updating eslint and the pluto eslint rules within all the packages seems to fix that getScope issue. I can create a PR for that.

Reply:
Oh that is way better 💯! You can also grab the changes from mine and open directly to main branch if you want!

Reply:
Cool! Updated https://github.com/Localitos/pluto/pull/877